libs/utils/decorate.js

```diff
@@ -126,17 +126,25 @@ export function getBlockSize(el, defaultSize = 1) {
 }
 
 export const decorateBlockHrs = (el) => {
-  const els = el.querySelectorAll('p');
+  const pTags = el.querySelectorAll('p');
   let hasHr = false;
-  [...els].forEach((e) => {
-    if (!e.textContent.startsWith('---')) return;
+  const decorateHr = ((tag) => {
+    const startWith = tag.textContent.trim().startsWith('---');
```

Comment:
Nit: Maybe a name more specific/meaningful.  What about`hr`?  Or, `hrTag` to follow suite of your `pTags` naming?